helm-chart/templates/09-worker-daemon-set.yaml

```diff
@@ -121,6 +121,14 @@ spec:
             value: '{{ (include "sentry.enabled" .) }}'
           - name: SENTRY_ENVIRONMENT
             value: '{{ .Values.tap.sentry.environment }}'
+          - name: PCAP_DUMP_ENABLE
```

Comment:
@bogdanvbalan I think this is redundant. You ended up using config-map. AFAIK environment variables are an alternative to using config map. I also, am not sure you're using these environment variables in the worker. LMK if you think I'm wrong.